templates/index.html

```diff
@@ -9,250 +9,266 @@
 
 {% block content %}
 	<!-- Hero banner -->
-	<section class="row row-hero">
-		<div class="inner-wrapper equal-height">
-			<div class="row-hero__text row-hero__col five-col">
-				<h2><img src="{{ ASSET_SERVER_URL }}33b96a7f-maas-logo.svg" alt="" class="row-hero__logo" itemprop="image" />Metal as a Service</h2>
-				<h1>Say hello to your physical cloud</h1>
-				<p class="row-hero__intro" itemprop="description">Complete automation of your physical servers for amazing data centre operational efficiency. On premises, open source and supported.</p>
-				<div class="row-hero__button-group">
-					<a href="/install" class="row-hero__button button--positive button--inline-positive" itemprop="url" >Install MAAS</a>
-					<a class="row-hero__button button--neutral row-hero__button--video button--inline-neutral venobox" data-type="youtube" data-autoplay="true" href="https://www.youtube.com/embed/eHfaiKlUYsM" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Video', 'eventAction' : 'Play', 'eventLabel' : 'Watch the video' });">
-						<span>Watch the video</span>
-					</a>
-				</div>
+  <section class="p-strip--image is-dark" style="overflow: hidden; background-color: #333; background-image:url('https://assets.ubuntu.com/v1/6e4b2b20-MAAS-hero-background.png')">
+		<div class="row u-equal-height u-vertically-center">
+			<div class="col-6">
+        <div class="p-heading-icon">
+          <div class="p-heading-icon__header">
+            <img class="p-heading-icon__img" src="{{ ASSET_SERVER_URL }}33b96a7f-maas-logo.svg" itemprop="image" />
+            <h1 class="p-heading--two p-heading-icon__title">Metal as a Service</h1>
+          </div>
+          <h2 class="p-heading--one">Say hello to your physical cloud</h2>
+          <p itemprop="description" class="p-heading--four">Complete automation of your physical servers for amazing data centre operational efficiency. On premises, open source andsupported.</p>
+        </div>
+        <ul class="p-inline-list">
+					<li class="p-inline-list__item"><a href="/install" class="p-button--positive" itemprop="url" >Install MAAS</a></li>
+					<li class="p-inline-list__item"><a class="p-button--neutral" data-type="youtube" data-autoplay="true" href="https://www.youtube.com/embed/eHfaiKlUYsM" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Video', 'eventAction' : 'Play', 'eventLabel' : 'Watch the video' });">
```

Comment:
Can we get an ```aria-label``` explaining what will happen when a user clicks on the video link.